src/Features/Core/Portable/NavigateTo/RoslynNavigateToItem.cs

```diff
@@ -243,14 +244,14 @@ private string ComputeSecondarySort()
                 int ComputeFolderDistance()
                 {
                     // No need to compute anything if there is no active document.  Consider all documents equal.
-                    if (_activeDocument == null)
+                    if (_activeDocument is not { } activeDocument)
```

Comment:
feels unnecessary right?  since we did a null check, the later lines below shoudl be safe to refer to _activeDocument, right?

(that's my personal preference, as i always find it harder to reason when there are two varaibles (with almost identical names) pointing to the same value in teh same scope.